db/schema.rb

```diff
@@ -15,6 +15,19 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "appointment", force: :cascade do |t|
+    t.date "date"
+    t.time "time"
+    t.boolean "pay", default: false, null: false
+    t.string "message"
+    t.bigint "patient_id"
+    t.bigint "psychologist_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["patient_id"], name: "index_appointment_on_patient_id"
+    t.index ["psychologist_id"], name: "index_appointment_on_psychologist_id"
+  end
+
```

Comment:
I think this is was added by mistake, this table was removed in this commit 
https://github.com/BC-MAY-21-ROR/proyecto-proy-t03/commit/c8d016ed7e2a72fec19170a29e7b25d6de9d38c6